extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessAction.scala

```diff
@@ -71,6 +71,8 @@ object ScenarioActionName {
   val Pause: ScenarioActionName     = ScenarioActionName("PAUSE") // TODO: To implement in future..
   val Rename: ScenarioActionName    = ScenarioActionName("RENAME")
 
+  val PerformSingleExecution: ScenarioActionName = ScenarioActionName("run now")
```

Comment:
Please add comment and/or TODO why we keep "run now".

Reply:
Comment added